### ISSUE TITLE
Seems all Unet param is trainable in the stage 2 training

### ISSUE THREAD

**Issue by zhoujiawei3** [OpenDriveLab/Vista#45]
It seems that with the vista_phase2_stage1.yaml, all the param is trainingable. 
I change the code in Vista/vwm/models/diffusion.py like this to test. And all the parameters are trainable.
```
def training_step(self, batch, batch_idx):
        loss, loss_dict = self.shared_step(batch)
        self.loss_avg = (self.loss_avg*self.loss_step + loss)/(self.loss_step+1)
        self.loss_step+=1
        self.log_dict(loss_dict, prog_bar=True, logger=True, on_step=True, on_epoch=True)
        trainable_list = []
        untrainable_list = []
        for name, param in self.model.named_parameters():
            if param.requires_grad:
                trainable_list.append(name)
            else:
                untrainable_list.append(name)
        with open('trainable.txt', 'w') as f:
            for name in trainable_list:
                f.write(name + '\n')
        with open('untrainable.txt', 'w') as f:
            for name in untrainable_list:
                f.write(name + '\n')
        print(f"loss_avg: {self.loss_avg}")
        self.log("global_step", self.global_step, prog_bar=True, logger=True, on_step=True, on_epoch=False)
        self.log("loss_avg", self.loss_avg, prog_bar=True, logger=True, on_step=True, on_epoch=False)
        if self.scheduler_config is not None:
            lr = self.optimizers().param_groups[0]["lr"]
            self.log("lr_abs", lr, prog_bar=True, logger=True, on_step=True, on_epoch=False)
        return loss
```
But in the paper, you say
 a![image](https://github.com/user-attachments/assets/5998656e-8066-4ccf-8c75-756aeef44a1a)
Maybe the config needs to be revised？

**Comment by Little-Podi** [comment]
Hi @zhoujiawei3. The `requires_grad` is not turned off due to the use of gradient checkpointing, but I didn't add the pretrained weights in the optimizer (please see our implementation [here](https://github.com/OpenDriveLab/Vista/blob/main/vwm/models/diffusion.py#L272-L277)). As a result, they will not be updated.

**Comment by zhoujiawei3** [comment]
Thanks for your reply！
